app/uk/gov/hmrc/tradergoodsprofilesrouter/service/RemoveRecordService.scala

```diff
@@ -38,7 +38,7 @@ class RemoveRecordService @Inject() (
     hc: HeaderCarrier
   ): Future[Either[EisHttpErrorResponse, Int]] = {
     val correlationId     = uuidService.uuid
-    val requestedDateTime = dateTimeService.timestamp.asStringSeconds
+    val requestedDateTime = dateTimeService.timestamp.asStringMilliSeconds
```

Comment:
Is this changing what goes into the header to EIS?

Reply:
nope, it is only used for audit request/response datetime